### DISCUSSION TITLE
Optimize markup to have more space for code

### DISCUSSION THREAD

**Discussion by vitaly-zdanevich** [community/community#5564] [Mobile]
![Screenshot_2021-09-09-20-43-08-242_com.github.android.jpg](https://user-images.githubusercontent.com/3514015/132736300-5836e7e9-b1f4-4e57-a166-9f68ee206006.jpg)

This screen shows ineffective UI - a lot of space wasted.

**Comment by aconsuegra** [comment]
Thanks for your feedback @vitaly-zdanevich! I'm going to pass it along to our design team, but in the meantime, do you have any suggestions about how to optimize this view to better utilize the screen space?

**Reply by vitaly-zdanevich** [reply]
I think

1) Add full screen mode, where we see only code.

2) Compact buttons on this screenshot - one line with controls would be good.

**Reply by aconsuegra** [reply]
Thanks for getting back with your suggestions! I'm going to track this internally and hopefully we can include some of your suggestions in the apps soon.
